### ISSUE TITLE
Git error and visual glitches. 

### ISSUE THREAD

**Issue by sandersantema** [jorgebucaran/fisher#415]
fish version: fish, version 2.7.1-1042-g99ecaec1
Mac OS 10.12.6

#### When trying to install something with fisher I get this: 
https://asciinema.org/a/L0RJWm6Be8JMAgbQLLUBDz3sc

Full error (since asciinema doesn't record scroll-back for some reason): https://gist.github.com/sandersantema/d6707c3968eedd353dd6d0741d714c40

#### Besides this fisher gives more errors which I can't figure out:
```
fisher update
find: ^: unknown primary or operator
find: ^: unknown primary or operator
OK fisherman is up to date
Done in 1s 311ms
```

#### Visual Glitches
Fisher also has some weird visual glitches when using the Alacritty terminal in combination with the solarized dark colour scheme, this isn't seen on the asciinema cast: 
<img width="541" alt="schermafbeelding 2018-04-11 om 15 27 46" src="https://user-images.githubusercontent.com/21057514/38619847-5c9dfafa-3d9d-11e8-9dbb-7e79120dd65d.png">

Thanks a lot in advance for looking into these problems.

**Comment by Markcial** [comment]
Better than an asciinema capture would be more insightful the logs resultin from a profiling session.
Execute a new terminal with the `fish -p <log-file>` command and check the `log-file` once the session ended.

In your case seems that the `find` command is receiving unexpected arguments. So if you can search for the calls to the find command maybe we can get to the problem.

Thanks

**Comment by jorgebucaran** [comment]
Fixed in [fisher/releases/tag/3.0.0](https://github.com/jorgebucaran/fisher/releases/tag/3.0.0)~! 🎉 

For more information see the [new documentation](https://github.com/jorgebucaran/fisher/blob/34b5257c5beaaf37f7d1988a6a9a62f8a0b1196c/README.md).

**Comment by jorgebucaran** [comment]
@sandersantema Fixed in [fisher/releases/tag/3.0.0](https://github.com/jorgebucaran/fisher/releases/tag/3.0.0)~! 🎉 

For more information see the [new documentation](https://github.com/jorgebucaran/fisher/blob/34b5257c5beaaf37f7d1988a6a9a62f8a0b1196c/README.md).
